src/MoBi.Presentation/DTO/IDescriptorConditionDTO.cs

```diff
@@ -42,6 +42,24 @@ internal class NotMatchConnditionDTO : IDescriptorConditionDTO
       public string TagDescription => AppConstants.NotMatch;
    }
 
+   internal class InContainerConditionDTO : IDescriptorConditionDTO
+   {
+      public string Tag { get; set; }
+
+      public TagType TagType => TagType.InContainer;
+
+      public string TagDescription => AppConstants.InContainer;
+   }
+
+   internal class NotInContainerConditionDTO : IDescriptorConditionDTO
```

Comment:
Not sure why we have all those DTOs. here. Probably one would do. I'll see if I can make it a bit easier to follow